crates/ruff/src/importer/mod.rs

```diff
@@ -75,6 +75,30 @@ impl<'a> Importer<'a> {
         }
     }
 
+    /// Move an existing import to the top-level, thereby making it available at runtime.
+    ///
+    /// If there are no existing imports, the new import will be added at the top
+    /// of the file. Otherwise, it will be added after the most recent top-level
+    /// import statement.
+    pub(crate) fn to_runtime_import(&self, import: &StmtImport, at: TextSize) -> Result<Edit> {
+        // Generate the modified import statement.
+        let content = autofix::codemods::retain_imports(
+            &[import.full_name],
+            import.stmt,
+            self.locator,
+            self.stylist,
+        )?;
+
+        Ok(if let Some(stmt) = self.preceding_import(at) {
+            // Insert after the last top-level import.
+            Insertion::end_of_statement(stmt, self.locator, self.stylist).into_edit(&content)
+        } else {
+            // Insert at the start of the file.
+            Insertion::start_of_file(self.python_ast, self.locator, self.stylist)
+                .into_edit(&content)
+        })
```

Comment:
Super Nit: 

To emphasize that the branching is to create different `insertions`. 

```suggestion
       let insertion = if let Some(stmt) = self.preceding_import(at) {
            // Insert after the last top-level import.
            Insertion::end_of_statement(stmt, self.locator, self.stylist)
        } else {
            // Insert at the start of the file.
            Insertion::start_of_file(self.python_ast, self.locator, self.stylist)
        };
        
        Ok(insertion.into_edit(&content))
```